### PR TITLE
jQuery deferreds

### PR DESCRIPTION
**Pull request by kswedberg**
This PR addresses https://github.com/jquery/learn.jquery.com/issues/701

### CODE REVIEW

File: page/code-organization/deferreds/jquery-deferreds.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ $.when(
 ).then( successFunc, failureFunc );
 ```
 
-The `$.when()` implementation offered in jQuery is quite interesting as it not only interprets deferred objects, but when passed arguments that are not deferreds, it treats these as if they were resolved deferreds and executes any callbacks (doneCallbacks) right away. It is also worth noting that jQuery's deferred implementation, in addition to exposing deferred.then(), a jQuery promise also supports the deferred.done() and deferred.fail() methods which can also be used to add callbacks to the deferred's queues.
+The `$.when()` implementation offered in jQuery is quite interesting as it not only interprets deferred objects, but when passed arguments that are not deferreds, it treats these as if they were resolved deferreds and executes any callbacks (doneCallbacks) right away. It is also worth noting that jQuery's deferred implementation, in addition to exposing deferred.then(), also supports the deferred.done() and deferred.fail() methods which can also be used to add callbacks to the deferred's queues.
```
Comment (AurelioDeRosa) [inline page/code-organization/deferreds/jquery-deferreds.md]: Minor point: doneCallbacks, deferred.then(),  deferred.done(), and deferred.fail() should be wrapped by the backtik.

File: page/code-organization/deferreds/jquery-deferreds.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ $.when(
...
+We will now take a look at a code example that uses many deferred features. This very basic application begins by consuming (1) an external news feed and (2) a reactions feed for pulling in the latest comments via `$.get()` (which will return a promise). When both requests are received, the `showAjaxedContent()` function that is called. The `showAjaxedContent()` function returns a promise that is resolved when animating both containers has completed. When the `showAjaxedContent()` promise is resolved, `removeActiveClass()` is called. The `removeActiveClass()` returns a promise that is resolved inside a `setTimeout()` after 4 seconds have elapsed. Finally, after the `removeActiveClass()` promise is resolved, the last `.then()` callback is called, provided no errors occurred along the way.
```
Comment (AurelioDeRosa) [inline page/code-organization/deferreds/jquery-deferreds.md]: Perhaps

> This very basic snippet

instead of

> This very basic application

?

Comment (AurelioDeRosa) [inline page/code-organization/deferreds/jquery-deferreds.md]: > via `$.get()` (which will return a promise)

should be

> via `$.get()` (which will return a promise-like object)

The reason is that it actually returns a [jqXHR object](http://api.jquery.com/Types/#jqXHR).

Comment (AurelioDeRosa) [inline page/code-organization/deferreds/jquery-deferreds.md]: I guess there is a typo in this sentence:

> the `showAjaxedContent()` function that is called.

It should be:

> the `showAjaxedContent()` function is called.

Comment (AurelioDeRosa) [inline page/code-organization/deferreds/jquery-deferreds.md]: Minor point:

> the last `.then()` callback is called

should  be

> the last `then()` callback is called

for consistency. Note the removed dot before the word "then"

**Comment by kswedberg** [comment]
Thanks a lot for the feedback!

**Comment by kswedberg** [comment]
Updated the PR, @AurelioDeRosa. Let me know if it looks good to you now.

**Comment by AurelioDeRosa** [comment]
LGTM

**Comment by kswedberg** [comment]
Closed by 9878874bfc350e44c2b8137cb3245e3ce4eb4a9b

**Pull request closed without merging**